demo.md

```diff
@@ -0,0 +1,99 @@
+# Armada Demo
+
+This guide will show you how to take a quick test drive of an Armada
+instance already deployed to AWS EKS.
+
+## EKS
+
+The EKS demo of Armada can be found at these URLs:
+- [API Server](https://armada.demo.armadaproject.io)
```

Comment:
We probably don't need this.  That link doesn't really provide anything useful so people will click and tell us the link is bad.  But not sure what the point of hyperlinking a api server without swagger docs?

Reply:
yes good point -- I'll change it in the other PR -- this file gets clobbered from master on every build

Reply:
well changed here, too, but master will always clobber